geoh5py/shared/entity.py

```diff
@@ -252,7 +250,7 @@ def get_entity_list(self, entity_type=ABC) -> list[str]:
         return sorted(name_list)
 
     @property
-    def metadata(self) -> str | dict | None:
+    def metadata(self):
```

Comment:
**question:** if not string nor dict or None, what else can it be ?
Better to update the type hint than removing it.

(same for the setter)

Reply:
Back, and done